css/findinpage.css

```diff
@@ -37,12 +37,6 @@ body.linux #findinpage-bar {
 	font-size: 0.85em;
 	vertical-align: middle;
 }
-#findinpage-bar .icon {
-	font-size: 1em;
-	padding: 0 0.5em;
-	margin-top: -2px;
-}
-#findinpage-bar .icon.fa-angle-up,
-#findinpage-bar .icon.fa-angle-down {
-	font-size: 1.2em;
-}
+#findinpage-bar button {
+	padding: 0 0.25em;
+}
```

Comment:
It would probably be nice if these were aligned better:

<img width="595" alt="Screen Shot 2020-07-16 at 10 59 57 AM" src="https://user-images.githubusercontent.com/10314059/87694251-925cf500-c753-11ea-9bab-c60b98053a04.png">

However, the old icons weren't aligned very well either, so it's fine if you want to leave it as-is.